volatility3/framework/symbols/linux/extensions/hash.py

```diff
@@ -0,0 +1,100 @@
+# This file is Copyright 2022 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+
+from typing import Dict, Tuple
+
+from volatility3.framework import exceptions, constants
+from volatility3.framework import objects, interfaces
+from volatility3.framework.objects import utility
+
+
+class bash_hash_table(objects.StructType):
+    def __init__(
```

Comment:
This doesn't *look* like the constructor does anything new/special that the super constructor wouldn't do.  Am I missing something here or is this unnecessary?